.github/dependabot.yml

```diff
@@ -0,0 +1,19 @@
+version: 2
+
+updates:
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      interval: "daily"
+    target-branch: master
+    reviewers:
+      - stephendolan
+
+  - package-ecosystem: npm
+    directory: "/src/browser-app-skeleton"
```

Comment:
I'm surprised at how this path is written. As opposed to being `src/browser-app-skeleton`. Does this automatically prepend `.` or something?

Reply:
Now you've got me wondering... their docs made me think the slash was necessary, but I've never not had something in the root directory to test it.

https://docs.github.com/en/free-pro-team@latest/github/administering-a-repository/configuration-options-for-dependency-updates#directory